### DISCUSSION TITLE
Does Milvus support suffix matching in boolean expression?

### DISCUSSION THREAD

**Discussion by intothephone** [milvus-io/milvus#24382] [Q&A and General discussion] [answered]
As the doc stated, it supports prefix matching when searching: VARCHAR like "prefix%". Can I use suffix matching?

VARCHAR like "%suffix"

Anyone tried it?

**Comment by yhmo** [comment, marked as answer]
Suffix and infix are not supported. The underlying index only supports prefix matching.

**Reply by intothephone** [reply]
Thanks for confirm.
